server/routes/checkAnswers.js

```diff
@@ -47,3 +150,64 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
 
   return router
 }
+
+// utility function - convert to string where input MAY be an array
+
+const convertNestedArraysToString = allElements => {
```

Comment:
Rather than creating a generic method for this - it might be worth making it very specific: 
e.g: `const getRestraintPositions = ({ restraintPositions }) => ...`

It's probably only worth trying to make a method generic if you have a few places where it's being used - this seems very specific to this one use case. 

(If you did need to create a generic method, then it should probably live with other more generic code (utils?) and be tested independently)

Reply:
I gave it a generic name on the chance that I may be able to use it elsewhere in the application. In the event, it hasn't been needed elsewhere. So yes, I agree that it can have a specific name.
Renamed and refactored.